Tests/LMStorageTests/LMCodableTests.swift

```diff
@@ -87,4 +87,19 @@ final class LMCodableTests: XCTestCase {
     XCTAssertEqual(movies.first?.title, "Black Panther")
     XCTAssertEqual(movies.last?.title, "Iron Man")
   }
+
+  func testParseNullable() {
+      let contet = ContentId(id: "Movie-ID", seriesTmdb: nil, source: "Amazon Prime")
+      let dict = contet.dictionaryNullable()
```

Comment:
@Paulohos I think this PR won't be required, you can use the double optional for the property you want to be encoded, even if it's null as described [here](https://stackoverflow.com/a/78528214/3683239).

**Example:**

<img width="290" alt="Screenshot 2025-06-17 at 12 44 40 PM" src="https://github.com/user-attachments/assets/24e8d31f-c61c-4e6f-8d6a-36bfdcafb357" />

<img width="521" alt="Screenshot 2025-06-17 at 12 44 23 PM" src="https://github.com/user-attachments/assets/1a541e13-bec8-401e-bacd-4f1853cc3f74" />

Reply:
Please let me know if it works for you